packages/cli/src/commands/setup/ui/libraries/mantine.js

```diff
@@ -74,6 +74,7 @@ export async function handler({ force, install, packages }) {
                 'web',
                 'add',
                 '-D',
+                '@emotion/react',
```

Comment:
I put it here because you need emotion react no matter what mantine packages you're using.